basics/components.md

```diff
@@ -12,15 +12,15 @@ Most of the components in GrimoireLab are written in Python, and provide Python
 
 The main components of GrimoireLab are:
 
-* **Perceval**. Includes components for retrieving data from data sources. All components provide an uniform API for accessing their data sources, based on Python generators and data dictionaries (JSON documents). [Learn more about Perceval](/grimoirelab-tutorial/perceval/intro.md).
-* **Graal**. leverages on the Git backend of Perceval and enhances it to set up ad-hoc source code analysis. It fetches the commits from a Git repository and provides a mechanism to plug third-party tools/libraries focused on source code analysis. [Learn more about Graal](/grimoirelab-tutorial/graal/intro.md).
+* **Perceval**. Includes components for retrieving data from data sources. All components provide an uniform API for accessing their data sources, based on Python generators and data dictionaries (JSON documents). [Learn more about Perceval](/grimoirelab-tutorial/perceval/intro.html).
+* **Graal**. leverages on the Git backend of Perceval and enhances it to set up ad-hoc source code analysis. It fetches the commits from a Git repository and provides a mechanism to plug third-party tools/libraries focused on source code analysis. [Learn more about Graal](/grimoirelab-tutorial/graal/intro.html).
 * **KingArthur**. Organizes the data retrieval from large collections of repositories of different kinds, providing a uniform API to all of them, and to the different retrieval jobs running in any moment. [Learn more about KingArthur](/grimoirelab-tutorial/basics/scenarios.md#arthur-orchestrating-data-retrieval).
```

Comment:
I found one more broken link here, it would be great if you can update this too.

```
- /scenarios.md#
+ /scenarios.html#
```

Reply:
Done